### PR TITLE
change cvss tables to make more sense

### PR DESCRIPTION
**Pull request by factoidforrest**
implements a new table for holding all settings for a given path glob, and for ranking those settings by precedence

### CODE REVIEW

File: lunatrace/bsl/hasura/migrations/lunatrace/1669838695525_change_cvss_tagging_table/up.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+DROP TABLE public.project_environmental_adjustment;
+
+
+CREATE TABLE public.project_folder_settings
+(
+    id         uuid PRIMARY KEY DEFAULT (gen_random_uuid()),
+    precedence INTEGER NOT NULL,
```
Comment (ajvpot) [inline lunatrace/bsl/hasura/migrations/lunatrace/1669838695525_change_cvss_tagging_table/up.sql:7]: We should add a comment that describes if lower or higher numbers are more preferred.

**Review by ajvpot — APPROVED** [review]

File: lunatrace/bsl/hasura/migrations/lunatrace/1669838695525_change_cvss_tagging_table/up.sql (hunk 1 above)
Comment (factoidforrest) [reply lunatrace/bsl/hasura/migrations/lunatrace/1669838695525_change_cvss_tagging_table/up.sql:7]: done

**Comment by factoidforrest** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by factoidforrest** [comment]
recheck

**Pull request merged**